project/BuildHelper.scala

```diff
@@ -224,7 +198,7 @@ object BuildHelper {
       "com.github.liancheng" %% "organize-imports" % "0.6.0",
       "com.github.vovapolu"  %% "scaluzzi"         % "0.1.23"
     ),
-    Test / parallelExecution := true,
+    Test / parallelExecution := scalaVersion.value != ScalaDotty,
```

Comment:
Have you checked whether this is still necessary? I suspect that lots of these settings were initially like this because of bugs/issues in the early days of Scala 3.